### PR TITLE
React UI: Added option to display shape text always

### PR DESCRIPTION
**Pull request by bsekachev**
Screenshot from 2020-04-02 22-15-42
Screenshot from 2020-04-02 22-15-49

### CODE REVIEW

File: cvat-ui/src/reducers/settings-reducer.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ const defaultState: SettingsState = {
         autoSave: false,
         autoSaveInterval: 15 * 60 * 1000,
         aamZoomMargin: 100,
+        showObjectsTextAlways: true,
```
Comment (ActiveChooN) [inline cvat-ui/src/reducers/settings-reducer.ts:31]: Why is it a good idea to enable this option by default?

Comment (bsekachev) [reply cvat-ui/src/reducers/settings-reducer.ts:31]: I did it for test. Forgot to disable.

File: cvat-canvas/README.md [hunk 2]
```diff
@@ -104,7 +108,9 @@ Canvas itself handles:
         dragCanvas(enable: boolean): void;
         zoomCanvas(enable: boolean): void;
 
+        mode(): void;
```
Comment (ActiveChooN) [inline cvat-canvas/README.md:111]: I see that `mode()` method returns Mode object

https://github.com/opencv/cvat/blob/feebec2670186f8f56f4314cb45d596d5dc13e8a/cvat-canvas/src/typescript/canvas.ts#L137-L139

Why did it change in documentation and Canvas interface?

Comment (bsekachev) [reply cvat-canvas/README.md:111]: It seems I did it accidently

**Pull request merged**